subprojects/docs/src/docs/release/notes.md

```diff
@@ -88,7 +88,7 @@ The `outgoingVariants` report now provides additional information that allows fu
 This report is useful when determining why a particular variant of this producer project was selected by another consumer project when the producer depends upon the consumer.  Run the report from the producer project, to list every variant built by it (including secondary variants only visible to other local projects).  The output contains the capabilities and attributes present on each variant of the producer, along with other information detailed below.  This output can be compared against the output of the new [`resolvableConfigurations` report](#resolvable-configurations) run in the consumer.
 
 
```

Comment:
@alllex Is the change to the image below intended?

Reply:
This is not a change to the image, rather a typo fix in the alt text right in the beginning of the line.
From
```
![`outoingVariants` output]
```
to
```
![`outgoingVariants` output]
```

I staged the docs locally and the image renders in the same way as before, so I think I didn't change any image content.

@ljacomet FYI

Reply:
Thanks, I missed that single letter